src/test/java/tech/jhipster/lite/module/domain/JHipsterModulesFixture.java

```diff
@@ -482,6 +536,9 @@ public static GradleProfilePlugin dockerGradlePluginDependency() {
       .id("com.bmuschko.docker-remote-api")
       .dependency(groupId("com.bmuschko"), artifactId("gradle-docker-plugin"))
       .versionSlug("docker-plugin")
+      .gradleImports()
```

Comment:
Let's simplify this, without `.gradleImports()` and just 
```suggestion
     .withBuildGradleImport("java.util.Properties")
     .withBuildGradleImport("java.util.foo")
```